dashboard/app/views/layouts/_unsupported_browser.html.haml

```diff
@@ -6,4 +6,5 @@
   #applab-unsupported-tablet.banner-text{style: 'display: none'}= t("compatibility.upgrade.applab_unsupported_tablet")
   #gamelab-unsupported-tablet.banner-text{style: 'display: none'}= t("compatibility.upgrade.gamelab_unsupported_tablet")
   #weblab-unsupported-local-storage.banner-text{style: 'display: none'}= t("compatibility.upgrade.weblab_unsupported_locale_storage")
+  #calc-eval-deprecated.banner-text{style: 'display: none'}!= t("deprecation.algebra_lab", bootstrap_algebra_url: 'https://www.bootstrapworld.org/materials/algebra/', cs_algebra_url: 'https://studio.code.org/s/algebra', markdown: true)
```

Comment:
This rung an old bell, reminded me of this thread I had with Elijah (2.5 years ago!):

https://codedotorg.slack.com/archives/C0T0UQH0R/p1594846686156200

Is unescaping HTML here necessary? I see its a copy of an existing pattern, but I think you might not need it?